docs/resource/download.en-US.md

```diff
@@ -1,7 +1,37 @@
 ---
 order: 0
-chinese: 资源下载
-english: Download
+title: Download
 ---
 
-TBD
+You can download design resources and tools about Ant Design in here. More design resource are still being collected.
+
+<div class="resource-cards">
+<a target="_blank" href="http://library.ant.design" class="resource-card">
+  <img src="https://os.alipayobjects.com/rmsportal/cnmjGfbBWUZPFiO.png">
+  <span class="resource-card-content">
+    <span class="resource-card-title">Ant Design Library v2.0.1</span>
+    <span class="resource-card-description">A powerful Axure library of Ant Design.</span>
+  </span>
+</a>
+<a target="_blank" href="http://ux.ant.design" class="resource-card">
+  <img src="https://os.alipayobjects.com/rmsportal/yfTqrQuSKcqBDLY.png">
+  <span class="resource-card-content">
+    <span class="resource-card-title">Ant UX</span>
+    <span class="resource-card-description">A series prototype library, help you to organize your page.</span>
+  </span>
+</a>
+<a target="_blank" href="https://github.com/ant-design/ant-design/releases/download/resource/iconfont-1.x.zip" class="resource-card">
+  <img src="https://os.alipayobjects.com/rmsportal/UEpOFKUQTZaUfnW.png">
+  <span class="resource-card-content">
+    <span class="resource-card-title">Web Font</span>
+    <span class="resource-card-description">Icon font package for your local reference.</span>
+  </span>
+</a>
+<a target="_blank" class="resource-card disabled">
+  <img src="https://os.alipayobjects.com/rmsportal/CxXbSpIXckHOtWl.png">
+  <span class="resource-card-content">
+    <span class="resource-card-title">Ant Cool</span>
+    <span class="resource-card-description">An Agile tool for sharing design resource.</span>
```

Comment:
<span class="resource-card-description">An Agile tool for sharing design resources.</span>